applications/FluidDynamicsApplication/custom_strategies/strategies/compressible_navier_stokes_explicit_solving_strategy_runge_kutta_4.h

```diff
@@ -197,6 +229,12 @@ class CompressibleNavierStokesExplicitSolvingStrategyRungeKutta4 : public Explic
         // Call the base RK4 finalize substep method
         BaseType::Initialize();
 
+        // Initialize the postprocess non-historical variables
+        for (auto &r_node : r_model_part.Nodes()) {
+            r_node.SetValue(MACH, 0.0);
+            r_node.SetValue(SOUND_VELOCITY, 0.0);
+        }
```

Comment:
parallel utils

Reply:
The initialization can't be in a parallel region. Note that this is done once to allocate memory.

Reply:
Actually this one deliberately can be done in parallel 
I can explain in detail in the next meeting

Reply:
OK. But how is the error in FullDebug avoided?

Reply:
Check closely, the error is only in the GetValue fct :) 

In a nutshell the idea is that if the user uses SetValue deliberately they know that it allocates
Since GetValue silently allocates (if not existing) there is an error in FullDebug

See the discussion in #3712

Reply:
Mmmm, I think I've experienced some issues with this un the past. In any case let's discuss It in the next meeting 👍

Reply:
OK. After going again through the Get/Set discussion I think you’re right. Thanks for reporting it¡¡

Reply:
Done in all the  `Initialize` methods involved.